xclim/indicators/atmos/_temperature.py

```diff
@@ -215,6 +216,21 @@ class TempWithIndexing(ResamplingIndicatorWithIndexing):
     compute=indices.heat_wave_frequency,
 )
 
+heat_wave_magnitude = Temp(
+    title="Heat wave magnitude",
+    identifier="heat_wave_magnitude",
+    units="degC",
```

Comment:
You are requesting degC here for the indicator, would that be the source of your issue ?

Reply:
That might be it indeed. What do you think should be the dedicated unit for it? "K d" like you suggested in the comment below (https://github.com/Ouranosinc/xclim/pull/1926#discussion_r1763248830)?

Reply:
As mentioned in this comment https://github.com/Ouranosinc/xclim/pull/1926#discussion_r1763282932 this issue is resolved by changing the units to "K d".